### ISSUE TITLE
[Bug]: Section heading should be "Border" instead of "Border & Shadow" for Modal widget

### ISSUE THREAD

**Issue by shadabbuchh** [appsmithorg/appsmith#18054]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Since we don't currently support modal shadows, the section heading for Modal widget should read only `Border` instead of `Border and Shadow`.

![Uploading Screenshot 2022-11-02 at 3.16.53 PM.png…]()

### Steps To Reproduce

1. D&D a Modal widget.
2. In the property pane go to the `Style` tab.

### Version

Cloud

**Comment by aswathkk** [comment]
We named this section to be "Border and Shadow" for the sake of consistency.
@dilippitchika Do you think we should change this to just "Border"?

**Comment by dilippitchika** [comment]
I don't think we need to update it for now. It might so happen that we later add shadow and then we have to change.
